### ISSUE TITLE
How to enable Fortify in CBA settings on server level?

### ISSUE THREAD

**Issue by celticalliance** [acemod/ACEX#137]
**Arma 3 Version:** `1.84` (stable)
**CBA Version:** `3.8.0` (stable)
**ACE3 Version:** `3.12.3` (stable)
**ACEX Version:** `3.3.1` (stable)

**Mods:**
```
- CBA_A3
- ace
- acex
```

**Description:**
- I was wondering how I can enable Fortify through the CBA settings on server level. We normally enable ACE and ACEX features through CBA settings on server level and therefore wish to avoid placing ACE and ACEX modules in our missions.

**Steps to reproduce:**
- None.

**Where did the issue occur?**
- Dedicated

**Placed Modules:**
- None.

**RPT log file:**
- None needed I suppose.

**Comment by celticalliance** [comment]
I just read in one of the closed issues that CBA settings at this point are not being considered for Fortify.

However, I think it would still be nice if you could just enable Fortify through CBA settings. I am asking because we are using our own custom training base and just want to be able to enable Fortify without having to put down all sorts of modules. I mean, there are so many things you can set up through CBA settings so why not this?

I am not sure how CBA settings work but in my opinion it would be quite easy to add or change several setting values to for example allow the CBA settings to enable Fortify and have several values for the Side setting that allows people to pick what sides can use Fortify.

Basically this is what I'm hoping to be able to. Yes, we are obviously running CBA and ACE + ACEX on server level but still we prefer to use CBA settings. So, it would be nice if this could be considered.

**Comment by Cuel** [comment]
Server admins can enable it on the fly using chat commands
```
#ace-fortify on
#ace-fortify off
```
